src/object/srv_obj_migrate.c

```diff
@@ -1137,7 +1137,7 @@ migrate_fetch_update_bulk(struct migrate_one *mrone, daos_handle_t oh,
 		return rc;
 	}
 
-	rc = bio_iod_prep(vos_ioh2desc(ioh));
+	rc = bio_iod_prep(vos_ioh2desc(ioh), BIO_CHK_TYPE_REBUILD);
```

Comment:
hmm, if bio_iod_prep can return extra information here to tell the DAM buffer is busy, then maybe rebuild can use normal buffer for fetch, then require the DMA buffer to update.

Reply:
"Normal buffer" consumes DRAM as well, why don't we configure proper DMA buffer upper limit to satisfy majority workload? For rare peak workload, I think we should just wait, instead of consuming more DRAM out of boundary.